Documentation/artifact-format.md

```diff
@@ -95,6 +89,38 @@ Currently there is only version 1, but this document may describe later versions
 if they are created.
 
 
+manifest
+----
+
+Format: text
+
+Contains the below content exactly:
+
+```
+Version: 2
```

Comment:
I've been thinking that we will use the same as artifact version, but probably it is redundant here. We are already having a version in `version` file.